app/src/main/java/com/github/multimatum_team/multimatum/AddDeadlineActivity.kt

```diff
@@ -0,0 +1,60 @@
+package com.github.multimatum_team.multimatum
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.View
+import android.widget.DatePicker
+import android.widget.TextView
+import android.widget.Toast
+import com.github.multimatum_team.multimatum.model.Deadline
+import com.github.multimatum_team.multimatum.model.DeadlineState
+import com.github.multimatum_team.multimatum.repository.DeadlineRepository
+import dagger.hilt.android.AndroidEntryPoint
+import kotlinx.coroutines.runBlocking
+import java.time.LocalDate
+import javax.inject.Inject
+
+/**
+ *  Activity who create a deadline using a DatePicker
+ */
+@AndroidEntryPoint
+class AddDeadlineActivity : AppCompatActivity() {
+    @Inject
+    lateinit var deadlineRepository: DeadlineRepository
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_add_deadline)
+
+        val datePicker = findViewById<DatePicker>(R.id.add_deadline_date_picker)
+        datePicker.init(2022, 1, 1, null)
```

Comment:
This is just a question, but why do you initialize the date picker to 1.01.2022? And I think that this "initialization date" should be a constant.

Reply:
This could perfectly be changed to initiate it on the actual date however I was waiting the implementation of the ClockService interface before doing so

Reply:
My bad, it is not necessary. Like CalendarView, we don't need to initiate it.

Reply:
Edit after some modification: 
It seems to be necessary to initiate it if we don't want to have some errors during the tests.
As I have not enough time, I will link this date next sprint to init the datePicker at the actual date